dv/management/commands/import_news.py

```diff
@@ -43,11 +43,14 @@ def _save(self, item):
             news.image = item['image'].replace('http://', 'https://')
             news.is_partnership = item['is_partnership'] == 'yes'
             if item['project_id']:
-                news.project_id = item['project_id'].strip()
+                project = Project.objects.filter(code=item['project_id']).exists()
+                if project:
+                    news.project_id = item['project_id'].strip()
             news.save()
 
-            for prg in item['programme_id'].split(', '):
-                news.programmes.add(prg.upper().strip())
+            if item["programme_id"]:
```

Comment:
Does this really suppress any errors? Without it, for an empty `item["programme_id"]`, we would iterate on `''.split(', ')` i.e. empty list, so I don't think it makes any difference.